starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,103 @@
+*{
+    font-family: Arial, Helvetica, sans-serif
+}
+
+/* Header Styling */
+
+.header{
+    background-image: url(/home/lossbow/Documents/Code/labs/lab-css-recipes-clone/starter-code/apple-pie-starter-code/images/apple-pie.jpg);
```

Comment:
can't open it :(

Reply:
what could I have done to avoid this? I initially tried ./images/apple-pie.jpg but it did not work...